db/academyPortal_2019-09-30.sql

```diff
@@ -115,6 +115,51 @@ VALUES
 UNLOCK TABLES;
 
 
+# Dump of table event_categories
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `event_categories`;
+
+CREATE TABLE `event_categories` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `event_categories` WRITE;
+/*!40000 ALTER TABLE `event_categories` DISABLE KEYS */;
+
+INSERT INTO `event_categories` (`id`, `name`)
+VALUES
+	(1,'Other'),
+	(2,'Sprint Review'),
+	(3,'Taster Session'),
+	(4,'Hiring Event');
+
+/*!40000 ALTER TABLE `event_categories` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table events
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `events`;
+
+CREATE TABLE `events` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL DEFAULT '',
+  `category` int(11) unsigned NOT NULL,
+  `date` date NOT NULL,
+  `start_time` time NOT NULL,
+  `end_time` time NOT NULL,
+  `notes` text,
+  PRIMARY KEY (`id`),
+  KEY `category` (`category`),
+  CONSTRAINT `events_ibfk_1` FOREIGN KEY (`category`) REFERENCES `event_categories` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
```

Comment:
Need to add a location field.

Reply:
'Location' is also missing from the getEvents method in the eventModel. This needs to be added.

Reply:
Have done this in separate fix branch